src/pages/iou/steps/MoneyRequestAmountForm.js

```diff
@@ -200,7 +200,7 @@ function MoneyRequestAmountForm({amount, currency, isEditing, forwardedRef, onCu
     const buttonText = isEditing ? translate('common.save') : translate('common.next');
 
     return (
-        <>
+        <ScrollView contentContainerStyle={styles.flexGrow1}>
```

Comment:
We need `contentContainerStyle={styles.flexGrow1}` because we want this page to use the whole vertically available space even if the space is larger than the page. (same behavior as seen on current **staging**, removing this would make the page not place the custom keyboard at the bottom of the screen).